modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -9,24 +9,18 @@ endif::[]
 You can see a history of changes you make to a project in Flow Designer.
 The saved states are listed by the ID of the user who made the change, time, date, and activity. The activity name is based on the action you performed at save time.
 
-Being able to view a saved state of a project is useful if you want to:
-
-* Undo an inadvertent change you made to a project by reverting back to a working state.
-* Review changes made by other users to the project.
-* Retrieve older values of some project configurations.
+Being able to restore a saved state of a project is useful if you want to:
 
+* Undo an inadvertent change you made to a project and revert to the working state.
+* Review changes made by other users on the project.
+* Retrieve previous values of connector configurations.
 
 == Procedure
 
-To restore a previous state of your project:
-
-. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas. 
-. Click a saved state from the list.
-. Inspect the saved state in the sidebar by clicking on an object in the flow. Use this method to inspect all of the objects in the flow.
-. Continue either selecting different saved states from the list or the objects within them until you locate the saved state you want to restore.
-. Click *Restore* at the top of the canvas.
-. If you want to restore the selected saved state, click *Restore* in the dialog. 
-
-The restored state is renamed and moved to the top of the list.
-
+To restore to a previous version of the project:
 
+. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas.
+. Click a saved state from the list. The saved state is a read-only view of each card in the project on the canvas.
+. Click on any card to see the details of its configuration. This can be helpful to review the values. 
+. After you select a state to revert your project to, click *Restore* at the top of the canvas. You see the conformation dialog.
```

Comment:
"You see the conformation dialog." sounds awkward to me. I'm used to stating such things like this: _The confirmation dialog appears._

Reply:
Yeah, think it's just a difference of writing styles. I was taught not to use: 'appears' because it does not translate well into Asian languages.

Reply:
Changed to your suggestion.